cmd/airplane/main.go

```diff
@@ -24,6 +27,22 @@ func main() {
 
 	cmd.Version = version
 
+	defer func() {
+		if r := recover(); r != nil {
+			logger.Log("Panic detected: %+v", r) // This does not print the stack trace.
```

Comment:
This will render in red text, similar to what we have below.

```suggestion
			logger.Error("The CLI unexpectedly crashed: %+v", r) // This does not print the stack trace.
```

Do you also want to report it via `analytics.ReportError`?

Reply:
I could be wrong, but I believe that Sentry already reports it in `Recover`: https://docs.sentry.io/platforms/go/panics/#usage